src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -75,9 +80,24 @@ export default class FiatConnectQuote extends NormalizedQuote {
     return fiatAccountToPaymentMethodMap[this.fiatAccountType]
   }
 
-  getFee(): number | null {
+  _getFee(): BigNumber | null {
     const feeString = this.quote.fiatAccount[this.fiatAccountType]?.fee
-    return feeString !== undefined ? parseFloat(feeString) : null
+    return feeString !== undefined ? new BigNumber(feeString) : null
+  }
+  // FiatConnect quotes denominate fees in fiat & crypto for CashIn & CashOut respectively
+  getFeeInCrypto(exchangeRates: { [token in Currency]: string | null }): BigNumber | null {
+    if (this.flow === CICOFlow.CashOut) {
+      return this._getFee()
+    }
+    return convertLocalAmountToCurrency(this._getFee(), exchangeRates[this.getCryptoType()])
```

Comment:
Is it possible for the local amount and fiat type to be different at all?

Reply:
No it is not. The code is pretty messy currently assuming that the local currency and the fiat type are the same across the flow. It'll take a good amount of work if we want to disentangle them